podcasts/EpisodeDetailViewController.swift

```diff
@@ -493,3 +493,11 @@ enum EpisodeDetailViewSource: String, AnalyticsDescribable {
 
     var analyticsDescription: String { rawValue }
 }
+
+// MARK: - Autoplay
+
+extension EpisodeDetailViewController: PlaylistAutoplay {
+    var playlist: EpisodesDataManager.Playlist {
+        .podcast(uuid: podcast.uuid)
```

Comment:
This logic is wrong. `EpisodeDetailViewController` can be opened from different places, I'm working on a fix.